volatility3/framework/symbols/windows/extensions/callbacks.py

```diff
@@ -14,7 +15,7 @@ class _SHUTDOWN_PACKET(objects.StructType, pool.ExecutiveObject):
     It exposes a function which sanity-checks structure members.
     """
 
-    def is_valid(self) -> bool:
+    def is_valid(self, type_map: Dict[int, str]) -> bool:
```

Comment:
This is a change in API and will require bumps somewhere.  Please could you consider a different name, or determining the type_map within the call please?

Reply:
Changed to `is_parseable(type_map)`